LICENSE

```diff
@@ -2,20 +2,19 @@ MIT License
 
```

Comment:
Since these are basically descriptions, more than code, wouldn't it better to use the CC license? (just wondering, please ignore if you already thought on this and decided this license was the way to go).

Reply:
We considered several licenses, including CC and discussed them with Mike Dolan and Kate Stewart from the Linux Foundation before the formation of CHAOSS. Our charter reads:
8.2.3. All contributions to implementation-agnostic metrics and standards, including associated scripts, SQL statements, and documentation, will be received and made available under the MIT License (https://opensource.org/licenses/MIT).

Reply:
OK, no problem then. I just didn't realized the wording of MIT license during those discussions.

Reply:
no worries :-)